modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -58,6 +58,10 @@ export interface CheckboxProps extends Themeable, React.InputHTMLAttributes<HTML
    * @default false
    */
   indeterminate?: boolean;
+  /**
+   * If set the Checkbox label may be truncated
+   */
+  width?: number;
```

Comment:
@ofalsap thanks for contributing! I'm wondering if we can change this prop into a boolean `isTruncated` that would basically apply the truncated styles. The width prop isn't as clear al though it achieves what you want to.

Reply:
In order to make it responsive maybe we can have `width:100%` Container will defining the width

Reply:
I think we may also want to support wrapping as well. What if we had something like:

`modules/common/react/lib/types.tsx` (since it will need to be used by `Radio` as well)
```tsx
export interface OverflowBehavior {
  overflow?: 'default' | 'wrap' | 'truncate'
}
```

`modules/checkbox/react/lib/Checkbox.tsx` (and `Radio`)
```tsx
export interface CheckboxProps extends OverflowBehavior, Themeable, React.InputHTMLAttributes<HTMLInputElement> 

// ...

({overflow}) => {
  if (overflow === 'wrap') {
    return { 
      wordWrap: 'break-word' 
    }
  }
  else if (overflow === 'truncate') {
    return {
      width: 100%,
      whiteSpace: 'nowrap', 
      overflow: 'hidden', 
      textOverflow: 'ellipsis'
    }
  }
}

Reply:
Hey guys! Thank you for the feedback! Will give it a go and update you again :)

Reply:
> @ofalsap thanks for contributing! I'm wondering if we can change this prop into a boolean `isTruncated` that would basically apply the truncated styles. The width prop isn't as clear al though it achieves what you want to.

Hey guys I tried doing it this way, instead of having width but with the width set to 100% the text does not truncate when I check it in story book? Do you guys mind providing advice on this? Thank you :)